hybris/tests/Makefile.am

```diff
@@ -140,13 +140,13 @@ test_camera_SOURCES = test_camera.c
 test_camera_CFLAGS = \
 	-I$(top_srcdir)/include \
 	$(ANDROID_HEADERS_CFLAGS)
-
```

Comment:
No need to remove this line.

Reply:
Other SOURCES/CFLAGS/LDADD variables are close together without blank lines between, shoudn't this be removed for consistency?

Reply:
Looking at other files there is not that much consistency in Makefiles, some have empty lines and others don't. Can be either way.